examples/grafana/README.md

```diff
@@ -1,57 +1,44 @@
-  Prometheus configuration.
- 
+  **Note:** If you are using the Prometheus server deployed by NGINX Service Mesh you can find the address by running [`nginx-meshctl config`](https://docs.nginx.com/nginx-service-mesh/reference/nginx-meshctl/#usage).
+  If you are using your own Prometheus server, make sure to add the NGINX Service Mesh [scrape config](../prometheus/README.md) to your Prometheus configuration.
 
-## Installing the NGINX Mesh Top Dashboard
+## Installing Example Dashboards
 
-To install the NGINX Mesh Top Dashboard using the Grafana UI follow these steps:
+To install the an example dashboard using the Grafana UI follow these steps:
 - Select the New Dashboard button and click Import.
-- Upload the [nginx-mesh-top.json](nginx-mesh-top.json) file or copy and paste the contents of the file in the textbox and click Load.
-- Select the Prometheus data source you configured previously from the dropdown
-  menu and click Import.
+- Upload the dashboard JSON, ex: [nginx-mesh-top.json](nginx-mesh-top.json), file or copy and paste the contents of the file in the textbox and click Load.
+- Select the Prometheus data source you configured previously from the dropdown menu and click Import.
 
 ![nginx-mesh-top](dashboard.png)
 
 
 ### Features
-NGINX Mesh Top dashboard includes the following stats and graphs:
+The default NGINX Mesh Top dashboard includes the following stats and graphs:
 
 - Stats:
-  - Global Success Rate: the percentage of requests that are successful over
-    the last 30 seconds (`nginxplus_upstream_server_responses` with response codes of `1xx` or `2xx`).
-  - Global Request Volume: the volume of requests over the last 30 seconds,
-    measured in requests per second (`nginxplus_http_total_requests`) . 
+  - Global Success Rate: the percentage of requests that are successful over the last 30 seconds (`nginxplus_upstream_server_responses` with response codes of `1xx` or `2xx`).
+  - Global Request Volume: the volume of requests over the last 30 seconds, measured in requests per second (`nginxplus_http_total_requests`) . 
   - Pods Monitored: The number of pods being scraped by Prometheus.
 - Graphs:
-  - Request Volume: this graph shows the volume of requests per Pod, in
-    requests per second (`nginxplus_http_total_requests`).
-  - Pod Success:  this graph shows the percentage of requests that are
-    successful per Pod (`nginxplus_upstream_server_responses` with response codes of
-    `1xx` or `2xx`).
+  - Request Volume: this graph shows the volume of requests per Pod, in requests per second (`nginxplus_http_total_requests`).
+  - Pod Success:  this graph shows the percentage of requests that are successful per Pod (`nginxplus_upstream_server_responses` with response codes of `1xx` or `2xx`).
   - Sidecar Memory Usage (RSS): this graph shows the Resident Set Size (RSS) of the NSM sidecars.
   - Sidecar Memory Usage (Private): this graph shows the Private memory used by the NSM sidecars.
 
```

Comment:
Should we outline the features of the summary dashboard as well?

Reply:
No, I'm fine just calling out the default as an example.